BigQuery/tests/Snippet/JobTest.php

```diff
@@ -99,13 +99,17 @@ public function testQueryResults()
         $this->assertInstanceOf(QueryResults::class, $res->returnVal());
     }
 
+    /**
+     * @doesNotPerformAssertions
```

Comment:
If a test does not make any assertions, it is marked risky. if no assertions are intended, it should be annotated as such. Being explicit makes it less likely that a test will "pass" despite not being correct due to its lacking any assertions.